test/support/factory.ex

```diff
@@ -0,0 +1,8 @@
+defmodule ElixirBoilerplate.Factory do
+  use ExMachina.Ecto, repo: ElixirBoilerplate.Repo
+
+  # This is a sample factory to make sure our setup is working correctly.
+  def name_factory(_) do
```

Comment:
No need for the arity 1. By default we should use arity 0. The arity 1 function is the "full control" mode https://github.com/thoughtbot/ex_machina/#full-control-of-factory

Reply:
No we have to use the arity 1 function because we don’t want to return a map. Factories with arity 0 functions must return a map.

Reply:
If we use an arity 0 function:

<img width="637" alt="" src="https://user-images.githubusercontent.com/11348/61409121-8c69d500-a8af-11e9-956d-9c6fbd7b2fa4.png">